### PR TITLE
Some parameter names and corrections

### PR DESCRIPTION
**Pull request by liach**
Fix the properties handler name conflict in #1145

Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

File: mappings/net/minecraft/server/dedicated/AbstractPropertiesHandler.mapping [hunk 1]
```diff
@@ -1,30 +1,91 @@
...
+		ARG 3 parsedTransformer
+		ARG 4 stringifier
+		ARG 5 fallback
 	METHOD method_16736 getDeprecatedBoolean (Ljava/lang/String;)Ljava/lang/Boolean;
-	METHOD method_16737 getString (Ljava/lang/String;Ljava/util/function/Function;Ljava/lang/Object;)Ljava/lang/Object;
+		ARG 1 key
+	METHOD method_16737 get (Ljava/lang/String;Ljava/util/function/Function;Ljava/lang/Object;)Ljava/lang/Object;
+		ARG 1 key
+		ARG 2 parser
+		ARG 3 fallbak
```
Comment (Juuxel) [inline mappings/net/minecraft/server/dedicated/AbstractPropertiesHandler.mapping]:
```suggestion
		ARG 3 fallback
```

File: mappings/net/minecraft/server/BannedPlayerEntry.mapping [hunk 2]
```diff
@@ -7,5 +9,5 @@ CLASS net/minecraft/class_3336 net/minecraft/server/BannedPlayerEntry
 		ARG 3 source
 		ARG 4 expiry
 		ARG 5 reason
-	METHOD method_14651 getProfileFromJson (Lcom/google/gson/JsonObject;)Lcom/mojang/authlib/GameProfile;
+	METHOD method_14651 profileFromJson (Lcom/google/gson/JsonObject;)Lcom/mojang/authlib/GameProfile;
```
Comment (Juuxel) [inline mappings/net/minecraft/server/BannedPlayerEntry.mapping:12]: Why not `get-`?

**Review by Juuxel — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/server/BannedPlayerEntry.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/server/BannedPlayerEntry.mapping:12]: this is converting this json object to a profile than getting a profile as part of this json object

**Review by Juuxel — APPROVED** [review]

File: mappings/net/minecraft/server/BannedPlayerEntry.mapping (hunk 2 above)
Comment (Runemoro) [reply mappings/net/minecraft/server/BannedPlayerEntry.mapping:12]: But if it's `getProfile(JsonObject)` it's clear it's getting it from JSON too.

File: mappings/net/minecraft/server/WhitelistEntry.mapping [hunk 3]
```diff
@@ -1,3 +1,7 @@
 CLASS net/minecraft/class_3340 net/minecraft/server/WhitelistEntry
-	METHOD method_14656 deserializeProfile (Lcom/google/gson/JsonObject;)Lcom/mojang/authlib/GameProfile;
+	METHOD <init> (Lcom/google/gson/JsonObject;)V
+		ARG 1 json
+	METHOD <init> (Lcom/mojang/authlib/GameProfile;)V
+		ARG 1 profile
+	METHOD method_14656 profileFromJson (Lcom/google/gson/JsonObject;)Lcom/mojang/authlib/GameProfile;
```
Comment (Runemoro) [inline mappings/net/minecraft/server/WhitelistEntry.mapping:6]: Should be `getProfile(JsonObject)` here too

File: mappings/net/minecraft/server/BannedPlayerEntry.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/server/BannedPlayerEntry.mapping:12]: nah getProfile is more like 
```json
{
  "profile": {(the_stuff_you_want_to_get)}
}
```
but actually it is
```json
{(the_stuff_you_want_to_get)}
```
So a better name is `profileFromJson` or `jsonToProfile`, whichever you like @Runemoro

**Review by Runemoro — APPROVED** [review]

**Pull request merged**